tests/capture-tests.js

```diff
@@ -243,20 +243,17 @@ require(["mobifyjs/utils", "capture"], function(Utils, Capture) {
         $("#qunit-fixture").append($iframe);
     });
 
-    asyncTest("ios8_0ScrollFix", function() {
+    // A metatag should not be inserted if it was not present in the original doc
```

Comment:
@jansepar looks like this test was essentially testing for the undesirable behaviour, i.e. the addition of the viewport metatag when one wasn't originally present. I'm reversing it to ensure that it _doesn't_ add a viewport mettag.